Examples/MotionManager/MotionManager/MotionManagerView.swift

```diff
@@ -117,15 +131,37 @@ func plot(buffer: [Double], scale: Double) -> Path {
 
 struct AppView_Previews: PreviewProvider {
   static var previews: some View {
-    AppView(
+    // Since MotionManager isn't usable in SwiftUI previews or simulators we create one that just
+    // sends a bunch of data on some sine curves.
+    var isStarted = false
+    let mockMotionClient = MotionClient(
```

Comment:
just a really simple mock motion client that sets up a timer to emit values, and does a simple `.filter` in order to control if it sends value out or not (so that we can start and stop it from the UI).